tools/autograd/templates/Functions.cpp

```diff
@@ -902,6 +902,40 @@ Tensor log_softmax_double_backward(const Tensor & grad, const Tensor & grad_outp
   return z * grad_output.sum(dim, true) * ((grad * z).sum(dim, true) - grad);
 }
 
+Tensor binary_cross_entropy_double_backward(const Tensor & grad_output, const Tensor & grad, const Tensor & input, const Tensor & target, const Tensor& weight, int64_t reduction) {
+  auto eps = 1e-12;
```

Comment:
I would have two questions here:
- Why `1e-12`?
- What is the type of eps? Are we sure it's not going to become `0`?

Reply:
Hey! 
1) I saw this epsilon value being used for calculating the first differential in the backward function, which is what I am doing here too so I just used the same value. 
2) backward function works fine with that eps, so I am assuming not?